Formula/postgresql.rb

```diff
@@ -98,6 +98,9 @@ def post_install
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
+      system "brew", "services", "start", "postgresql"
```

Comment:
I figured something out, perhaps we could do something like :
`system "#{bin}/initdb", "#{var}/postgres"`
`system "pg_ctl -D #{var}/postgres -l #{var}/postgres/server.log start"`
`sleep(0.5)`
`system "createdb"`
`system "pg_ctl -D #{var}/postgres stop -s -m fast"`
I think so because
- `initdb` works fine
- `pg_ctl` command is for starting the postgresql server (does the job of brew services start)
- `sleep` is required because the server actually takes time to start, maybe due to a process which continues even after the system call ( I tested a variant without sleep, and the script couldn't detect the started server. Though running each command individually did the trick. )
- Next `pg_ctl` is used to stop the server